vmdk_plugin/drivers/vmdk/vmdkops/esx_vmdkcmd.go

```diff
@@ -44,16 +44,18 @@ type EsxVmdkCmd struct {
 }
 
 const (
-	commBackendName        string = "vsocket"
-	maxRetryCount                 = 5
-	protocolCurrentVersion        = "2"
+	commBackendName string = "vsocket"
+	maxRetryCount          = 5
+	// Server side understand protocol version. If you are changing client/server protocol we use
+	// over VMCI, PLEASE DO NOT FORGET TO CHANGE IT FOR SERVER in file <vmdk_ops.py> !
+	protocolCurrentVersion = "2"
```

Comment:
I do agree with @shaominchen , calling clientProtocolVersion makes it much easier to comprehend